plugin/web/extensions/sidebar/modals/datacatalog/components/content/YourDataPage/LocalDataTab.tsx

```diff
@@ -0,0 +1,88 @@
+import { CatalogItem, convertRaw } from "@web/extensions/sidebar/core/processCatalog";
+import { Form } from "@web/sharedComponents";
+import { InboxOutlined } from "@web/sharedComponents/Icon/icons";
+import Upload, { UploadProps, UploadFile } from "@web/sharedComponents/Upload";
+import { RcFile } from "antd/lib/upload";
+import { useCallback, useState } from "react";
+
+import FileTypeSelect from "./FileTypeSelect";
+
+type Props = {
+  onOpenDetails?: (data?: CatalogItem) => void;
+};
+
+const LocalDataTab: React.FC<Props> = ({ onOpenDetails }) => {
+  const [fileList, setFileList] = useState<UploadFile[]>([]);
+  const fileFormats = ".kml,.czml,.topojson,.geojson,.json,.gltf,.glb";
+
+  const onRemove = useCallback((_file: UploadFile) => {
+    setFileList([]);
+  }, []);
+
+  const beforeUpload = useCallback(
+    (file: RcFile, files: RcFile[]) => {
+      // Catalog Item
+      const filename = file.name;
+      const id = "id" + Math.random().toString(16).slice(2);
+      const url = URL.createObjectURL(file);
+      const item = {
+        id: id,
+        description:
+          "This file only exists in your browser. To share it, you must load it onto a public web server.",
```

Comment:
Can you find a way to check plateau v1's Japanese for this line? Or ask a Japanese member to translate?